ojd_daps_skills/analysis/OJO/streamlit_viz/process_viz_data.py

```diff
@@ -289,6 +410,7 @@ def get_top_skills_per_group(percentage_group_skills_df, esco_code2name, top_n=2
     )
 
     percentage_sector_skill_by_group_list = []
+    percentage_sector_skill_by_group_list_no_trans = []
```

Comment:
@india-kerle in the same way as this has collected the "not transversal" top skills, there should be a modification to also collect and save out "just transversal".